sql/src/test/resources/calcite/tests/window/range_handling.sqlTest

```diff
@@ -1,24 +0,0 @@
-type: "operatorValidation"
-
-sql: |
-  SELECT
-    FLOOR(m1/3),
-    DENSE_RANK() OVER (ORDER BY FLOOR(m1/3)),
-    'prefix',
-    COUNT(1) OVER (ORDER BY FLOOR(m1/3) ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW),
-    COUNT(1) OVER (ORDER BY FLOOR(m1/3) RANGE BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW),
-    'postfix',
-    COUNT(1) OVER (ORDER BY FLOOR(m1/3) ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING),
-    COUNT(1) OVER (ORDER BY FLOOR(m1/3) RANGE BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING),
-    'k(1)',
-    COUNT(1) OVER (ORDER BY FLOOR(m1/3) ROWS BETWEEN 1 PRECEDING AND 1 FOLLOWING),
-    COUNT(1) OVER (ORDER BY FLOOR(m1/3) RANGE BETWEEN 1 PRECEDING AND 1 FOLLOWING)
```

Comment:
why remove this test; and not just these?

Reply:
yeah, make sense to just remove these 1 PRECEDING AND 1 FOLLOWING. Later on when we start supporting them, we can modify this test to keep the comparisons between ROWS and RANGE in different scenarios at one place. will update it. Thanks for the review @kgyrtkirk !!